config/application.rb

```diff
@@ -170,7 +170,8 @@ class Application < Rails::Application
       :app => {:enabled => true},
       :ldap => {:enabled => false},
       :permissions => {:enabled => false},
-      :sql => {:enabled => false}
+      :sql => {:enabled => false},
+      :audit => {:enabled => true}
```

Comment:
Well, I am hoping to create a dedicated logger for all audit logs that are too noisy for audited gem. Fact importer is just the first thing here. For the same reason I named the initializer as `audit_*.rb`.